library/core/src/array/mod.rs

```diff
@@ -43,7 +43,8 @@ mod iter;
 #[cfg(not(feature = "ferrocene_certified"))]
 pub(crate) use drain::drain_array_with;
 #[stable(feature = "array_value_iter", since = "1.51.0")]
-#[cfg(not(feature = "ferrocene_certified"))] /* blocked by IntoIter */
+// blocked by IntoIter
+#[cfg(not(feature = "ferrocene_certified"))]
```

Comment:
This re-formatting should not be necessary. In the past the previous comment style was accepted by CI.

Reply:
It is necessary since it's not detected by the script if we do

```rust
#[cfg(not(feature = "ferrocene_certified"))] // Blah blah
mod Foo {}
```

Reply:
Ah true!